nixos/modules/services/networking/firewall.nix

```diff
@@ -123,7 +123,7 @@ let
       # Perform a reverse-path test to refuse spoofers
       # For now, we just drop, as the raw table doesn't have a log-refuse yet
```

Comment:
This comment is out-of-date if you check line 134.

Reply:
Yeah, I thought of getting rid of it too -- will do.

Reply:
It is a bit strange why nixos does rpfilter with netfilter since Linux already check reverse path with `net.ipv4.conf.all.rp_filter`

Reply:
cc @andir

Reply:
IMO this is fine. Using `validmark` shouldn't cause any (additional) trouble.

I don't think it is an issue using the rpfilter mod in conjunction with the sysctl. It is the only option for IPv6 anyway.